recipes/cupy-xarray/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "cupy-xarray" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cupy-xarray-{{ version }}.tar.gz
+  sha256: 3162da1552182d471b0d11bec1779cd1ad49910c973827b643efa3213e611012
```

(to go with the version update above)

```suggestion
  sha256: fcaf71201fa75bfbb6008740787ee0c34b1ad83d7ca381e2dd915a5e6af7b3c6
```